yt_dlp/extractor/bilibili.py

```diff
@@ -51,16 +51,21 @@ def extract_formats(self, play_info):
         flac_audio = traverse_obj(play_info, ('dash', 'flac', 'audio'))
         if flac_audio:
             audios.append(flac_audio)
+        # dolby audio (if exists)
+        audios.extend(traverse_obj(play_info, ('dash', 'dolby', 'audio', ...)))
         formats = [{
             'url': traverse_obj(audio, 'baseUrl', 'base_url', 'url'),
             'ext': mimetype2ext(traverse_obj(audio, 'mimeType', 'mime_type')),
-            'acodec': audio.get('codecs'),
+            'acodec': (audio.get('codecs') or '').lower() or None,
```

Comment:
use traversal here to be safe IMO
```suggestion
            'acodec': traverse_obj(audio, ('codecs', {str.lower})),
```